schema_editor/app/scripts/schemas/schemas-service.js

```diff
@@ -0,0 +1,27 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Schemas() {
+        var module = {
+            Object: Object
+        };
+        return module;
+
+        function Object() {
+            return {
+                type: 'object',
+                title: '',
+                plural_title: '',
```

Comment:
Side note: I wonder if we should try and do something slick with DRF serializers to remove the need for underscores in our js.